### DISCUSSION TITLE
Windows 2016 data center Seq not loggin data

### DISCUSSION THREAD

**Discussion by jfranklyn** [datalust/seq-tickets#1182] [Help]
Hello all,
I have Seq version 6.0.5595.0 installed on 2 Azure VM's running Windows 2016 data center. When I browse the Seq website the page displays, but it is blank. I have other app servers pointer to this Seq server, so it should be logging messages, but it is not. The display is blank. I've restarted the Seq service and ran the administrator to re-configure the app. I don't see any errors in the windows or system event logs. I've looked in the c:\programdata\seq\log folder and don't see anything that looks like an error in the latest log file. We have other Seq servers that are working with no problems.
Any help would be greatly appreciated.

**Comment by nblumhardt** [comment]
Hi John,

Thanks for getting in touch. Did the same servers previously show data and break after an upgrade, or are they being newly set up?

Are there any errors shown in the _Data > Ingestion_ page? (There should be an ingestion log link there, if so.)

Best regards,
Nick

**Comment by jfranklyn** [comment]
Hello,
These servers are both new seq installs and have never worked. I've uploaded the latest seq log file. The attached image is all the I see when I browse the seq website. Looks almost like seq did not install properly, but the administrator works with no errors.

![image](https://user-images.githubusercontent.com/59396137/113217999-65e6fb80-9244-11eb-8761-1050197d872f.png)
[seq-20210331.log](https://github.com/datalust/seq-tickets/files/6239846/seq-20210331.log)

**Reply by nblumhardt** [reply]
Thanks for your reply, John.

Looks like you may be using IE, which isn't supported as of Seq 2021.1; there's definitely data being ingested, so you'll more than likely see it when you switch browsers - does this help?

Best regards,
Nick

**Comment by jfranklyn** [comment]
that makes sense, we would rather use Chrome, so I'll install it on these servers. Thanks for all your help.

**Reply by nblumhardt** [reply]
That's great, thanks John.
